packages/feedback/src/app/modules/feedback/feedback-routing.module.ts

```diff
@@ -0,0 +1,13 @@
+import { NgModule } from '@angular/core';
+import { Routes, RouterModule } from '@angular/router';
+
+const routes: Routes = [
+  {
+    
```

Comment:
I think all the routes for a small SPA such as this one should be written inside the `app-routing.module.ts`. And a sub-module should only be used for a complex application with multiple sub-modules.

Reply:
so you mean in this case no need of  **feedback-routing.module.ts**   and i can write all the route inside **app-routing-modules.ts**   ?